crates/aggchain-proof-builder/Cargo.toml

```
@@ -5,10 +5,32 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
+aggchain-proof-core.workspace = true
+aggkit-prover-config.workspace = true
+aggkit-prover-types.workspace = true
+alloy = { version = '0.11.0', default-features = false, features = [
+    "consensus",
+    "reqwest",
+    "json-rpc",
+    "rlp",
+    "rpc",
+    "rpc-client",
+    "rpc-types-eth",
+    "rpc-types-trace",
+    "providers",
+    "transports",
+    "transport-http",
+    "rpc-types-debug",
+] }
+
+anyhow.workspace = true
 futures.workspace = true
 thiserror.workspace = true
 tower = { workspace = true, features = ["timeout"] }
 tracing.workspace = true
+serde = { workspace = true, features = ["derive"] }
+sp1-sdk = { workspace = true, features = ["native-gnark"] }
+url = "2.5.4"
```

`url` is already in the workspace. Also, why not just use `alloy` from the workspace, that already requires `"full"` anyway? This way we won't have to care about the exact feature set here, which isn't checked in CI because CI runs workspace tests, which unifies feature sets anyway.

---

```suggestion
url.workspace = true
```

Is this not what you were suggesting @Ekleog-Polygon ?

---

Right! When doing my re-review I had missed that this one was still missing :)